packages/relay/tests/lib/eth/eth_getLogs.spec.ts

```diff
@@ -582,4 +583,13 @@ describe('@ethGetLogs using MirrorNode', async function () {
     expectLogData1(result[0]);
     expectLogData2(result[1]);
   });
+
+  it('Should return empty log if address = ZeroAddress', async () => {
+    restMock.onGet(BLOCKS_LIMIT_ORDER_URL).reply(200, { blocks: [latestBlock] });
+    restMock.onGet('blocks/0').reply(200, DEFAULT_BLOCK);
+    restMock.onGet('blocks/latest').reply(200, DEFAULT_BLOCK);
+    const result = await ethImpl.getLogs(null, '0x0', 'latest', ethers.ZeroAddress, DEFAULT_LOG_TOPICS);
+    expect(result).to.exist;
```

Comment:
Nit. This would be more precise and readable:

`expect(results).to.deep.equal([]);`